docassemble/AssemblyLine/data/questions/al_saved_sessions.yml

```diff
@@ -0,0 +1,264 @@
+---
+variable name: al_sessions_url_ask_snapshot
+data:
+  - undefine: 
+    - al_sessions_save_session_snapshot
+    - al_sessions_new_session_id
+  - al_sessions_snapshot_label
+  - al_sessions_save_session_snapshot
+  - al_sessions_snapshot_success
```

Comment:
Minor: the `al_sessions_snapshot_success` screen isn't always a success, it's more like "It worked!" or "It failed", I'd change the name to reflect that, maybe `al_sessions_snapshot_finished`?